pkg/steps/index_generator_test.go

```diff
@@ -117,3 +118,61 @@ COPY --from=builder /database/ database`,
 		})
 	}
 }
+
+func TestValidateIndexGeneratorStep(t *testing.T) {
```

Comment:
I guess this should be called
```suggestion
func TestDatabaseIndex(t *testing.T) {
```